packages/js/data/src/shipping-zones/README.md

```diff
@@ -13,20 +13,19 @@ Example:
 
```

Comment:
I think it'll be good update here as well: https://github.com/woocommerce/woocommerce/blob/3b92b879c7ede470b2f531189389424340f3d417/packages/js/data/src/shipping-zones/README.md?plain=1#L10

Reply:
Thanks, updated it in a0dfef5f1d74e5a9ebc39ae7345cbd93748256b9! 🙏